editor/editor_help.cpp

```diff
@@ -2179,6 +2180,10 @@ void EditorHelp::_wait_for_thread() {
 void EditorHelp::_gen_doc_thread(void *p_udata) {
 	DocTools compdoc;
 	compdoc.load_compressed(_doc_data_compressed, _doc_data_compressed_size, _doc_data_uncompressed_size);
+	const Vector<Engine::ExtensionClassDoc> &ext_doc = Engine::get_singleton()->get_extensions_class_doc();
+	for (int i = 0; i < ext_doc.size(); i++) {
+		compdoc.load_compressed(ext_doc[i].compressed_data.ptr(), ext_doc[i].compressed_size, ext_doc[i].uncompressed_size);
+	}
```

Comment:
Seems like this approach replaces the original `_doc_data_compressed` with the extension doc. So we'd need to add something like `append_compressed` that doesn't discard existing DocTools state.

Reply:
> Or an extra argument for `load_compressed` to skip `class_list.clear();` line, it should be the only change that's necessary.

Makes sense, did that in the end.